src/qibolab/pulses.py

```diff
@@ -101,6 +102,7 @@ class PulseShape(ABC):
         pulse (Pulse): the pulse associated with it. Its parameters are used to generate pulse waveforms.
     """
 
+    SAMPLING_RATE = 1e9  # 1e9  # 1GSaPS
     SAMPLING_RATE = 1e9  # 1GSaPS
```

Comment:
Duplicated line

Reply:
I will create an issue as Sampling_Rate should not be hardcoded for qblox